Build/Scripts/BuildLinux.js

```diff
@@ -23,6 +23,9 @@ function copyAtomicNET() {
 
 function copyAtomicEditor() {
 
+    if ( config["noclean"])
+        host.cleanCreateDir(editorAppFolder);
```

Comment:
The noclean option is primarily intended for building the binaries, in general probably want a clean folder for the app copy.  In either case, should this be ```if ( !config["noclean"] )``` ?